libs/wingc/src/jsify.rs

```diff
@@ -1286,6 +1286,7 @@ impl<'a> JSifier<'a> {
 		&self,
 		class: Option<&AstClass>,
 		func_def: &FunctionDefinition,
+		is_closure: bool,
```

Comment:
nit: should a jsify test be added as well?

Reply:
How do you mean?

Reply:
Adding a case for this fix in the jsify snapshot tests, the ones in `wingc/jsify/tests.rs`